lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -487,12 +487,27 @@ export abstract class BaseQueryBuilderService {
         let query = '';
 
         this.categories.forEach((facet) => {
-            const customQuery = this.queryFragments[facet.id];
-            if (customQuery) {
-                if (query.length > 0) {
-                    query += ' AND ';
+            let customQuery: string;
+
+            if (facet.component.selector === 'widget-composite') {
+                const widgetTabs: any[] = facet.component.settings.tabs;
```

Comment:
can we use something different than any?

Reply:
This is dead code that was used in an older approach. Reverted this change back to its original form